src/react/src/__tests__/components/ClaimAttachmentsUploader.test.js

```diff
@@ -16,110 +15,71 @@ class File {
-        fireEvent.change(fileInputField, { target: { files: fileList } });
-        const actions = initialState.getActions();
-
-        // Test changes in real application store given action data we triggered
-        store.dispatch({
-            type: actions[0].type,
-            payload: actions[0].payload,
-            error: false
-        })
-
-        const updatedUploadFiles = store.getState().claimFacility.claimData.formData.uploadFiles
-        expect(updatedUploadFiles.length).toBe(2)
-    });
+    it('adds files by action', () => {
+        const {reduxStore} = renderWithProviders(
+            <ClaimAttachmentsUploader inputId="files" files={[]} updateUploadFiles={() => {}}/>
+        )
+
+        reduxStore.dispatch(
+            updateClaimAFacilityUploadFiles([new File(['file contents'], 'attachment_1.jpg', { type: 'image/jpg' })])
+        )
 
-    it('removes files when clicked', async () => {
-        const handleRemoveFile = jest.fn()
+        const updatedUploadFiles = reduxStore.getState().claimFacility.claimData.formData.uploadFiles
+        expect(updatedUploadFiles.length).toBe(1)
+    });
 
-        const { getByTestId } = render(
-            <Provider store={initialState}>
-                <ClaimAttachmentsUploader handleRemoveFile={handleRemoveFile} />
-            </Provider>)
+    it('removes files by action', async () => {
+        const {reduxStore} = renderWithProviders(
+            <ClaimAttachmentsUploader inputId="files" files={[]} updateUploadFiles={() => {}}/>
+        )
 
-        const removeFileButton = getByTestId('claim-attachments-uploader-remove');
-        fireEvent.click(removeFileButton);
+        reduxStore.dispatch(
+            updateClaimAFacilityUploadFiles([new File(['file contents'], 'attachment_1.jpg', { type: 'image/jpg' })])
+        )
 
-        const actions = initialState.getActions();
+        const updatedUploadFilesFirst = reduxStore.getState().claimFacility.claimData.formData.uploadFiles
+        expect(updatedUploadFilesFirst.length).toBe(1)
 
-        store.dispatch({
-            type: actions[0].type,
-            payload: actions[0].payload,
-            error: false
-        })
+        reduxStore.dispatch(
+            updateClaimAFacilityUploadFiles([])
+        )
 
-        const updatedUploadFiles = store.getState().claimFacility.claimData.formData.uploadFiles
-        expect(updatedUploadFiles.length).toBe(0)
+        const updatedUploadFilesSecond = reduxStore.getState().claimFacility.claimData.formData.uploadFiles
+        expect(updatedUploadFilesSecond.length).toBe(0)
     });
 
     it('displays error message for unsupported file format', () => {
-        const { getByTestId, getByText } = render(<ClaimAttachmentsUploader store={initialState} />);
+        const { getByTestId, getByText } = renderWithProviders(
+            <ClaimAttachmentsUploader inputId="files" files={[]} updateUploadFiles={() => {}}/>
```

Comment:
You propably don't need `updateUploadFiles={() => {}}` here, but if this prop will obligated in the proptypes, then keep it.